etsi_its_conversion/etsi_its_conversion/src/Converter.cpp

```diff
@@ -204,10 +204,39 @@ void Converter::setup() {
     subscriber_udp_ = this->create_subscription<UdpPacket>(kInputTopicUdp, subscriber_queue_size_, std::bind(&Converter::udpCallback, this, std::placeholders::_1), subscriber_options);
   }
   if (std::find(udp2ros_etsi_types_.begin(), udp2ros_etsi_types_.end(), "cam") != udp2ros_etsi_types_.end()) {
+    convert_udp_to_cam_service_ = this->create_service<conversion_srv::ConvertUdpToCam>(
+        "~/udp_to_cam",
```

Comment:
move definition of all service names to the top where topics are defined, e.g., `kInputTopicCam `